tests/test_toolsqa_elements_zr.py

```diff
@@ -0,0 +1,175 @@
+import unittest
+from selenium.webdriver.support.wait import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.common.by import By
+from utils.base_test import BaseTest
+from selenium.webdriver import ActionChains
+import os
+import time
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+
+BASE_URL = 'https://demoqa.com/elements'
+TEXT_BOX_URL = 'https://demoqa.com/text-box'
+CHECKBOX_URL = 'https://demoqa.com/checkbox'
+RADIOBUTTON_URL = 'https://demoqa.com/radio-button'
+WEBTABLES_URL = 'https://demoqa.com/webtables'
+BUTTONS_URL = 'https://demoqa.com/buttons'
+LINKS_URL = 'https://demoqa.com/links'
+DOWNLOAD_URL = 'https://demoqa.com/upload-download'
+
```

Comment:
BASE_URL better to use to keep domain name, and combination with other links:

> BASE_URL = 'https://demoqa.com'
ELEMENTS_URL = f'{BASE_URL}/elements'
TEXT_BOX_URL = f'{BASE_URL}/text-box'
CHECKBOX_URL = f'{BASE_URL}/checkbox'
RADIO_BUTTON_URL = f'{BASE_URL}/radio-button'
WEB_TABLES_URL = f'{BASE_URL}/webtables'
BUTTONS_URL = f'{BASE_URL}/buttons'
LINKS_URL = f'{BASE_URL}/links'
DOWNLOAD_URL = f'{BASE_URL}/upload-download'

Reply:
done